nuto/mechanics/mesh/MeshGmsh.cpp

```diff
@@ -360,8 +361,12 @@ const NuTo::InterpolationSimple& CreateElementInterpolation(NuTo::MeshFem& rMesh
         return rMesh.CreateInterpolation(InterpolationTrussLobatto(2));
     case 9:
         return rMesh.CreateInterpolation(InterpolationTriangleQuadratic());
+    case 10:
+        return rMesh.CreateInterpolation(InterpolationQuadLobatto(2));
     case 11:
         return rMesh.CreateInterpolation(InterpolationTetrahedronQuadratic());
+    case 12:
+        return rMesh.CreateInterpolation(InterpolationBrickLobatto(2));
```

Comment:
We probably should start using the interpolation factory here, than we would have gotten almost that "for free".